services-turf/src/test/java/com/mapbox/turf/TurfMiscTest.java

```diff
@@ -490,4 +493,27 @@ public void testShortLine() throws IOException, TurfException {
     assertEquals(sliced.coordinates().get(sliced.coordinates().size() - 1).coordinates(),
       end_point.coordinates());
   }
+
+  @Test
+  public void testLineIntersects() throws
+          NoSuchMethodException, SecurityException,
+          IllegalAccessException, IllegalArgumentException, InvocationTargetException {
+    Method method = TurfMisc.class.getDeclaredMethod("lineIntersects",
```

Comment:
I'd like to suggest testing against the public API instead of using reflections.